ext/node/polyfills/internal_binding/stream_wrap.ts

```diff
@@ -341,6 +356,8 @@ export class LibuvStreamWrap extends HandleWrap {
         return this.#read();
       }
 
+      if (e.message === "cancelled") return null;
```

Comment:
😢 can we use `Deno.errors.<something>` here? This seems super brittle

Reply:
We can't use `Deno.errors.Interrupted` which means "operation was cancelled for user" whereas this one means "operation was cancelled for readStop()". Since this isn't a public API error I think a custom generic error is probably fine and faster to create.